### ISSUE TITLE
Add redbubble link to README

### ISSUE THREAD

**Issue by burgerga** [rstudio/hex-stickers#10]
I noticed that the README says you can't buy individual stickers, but I bought some at https://www.redbubble.com/people/rstudio-inc

If that is the official place to buy stickers, maybe include it in the README?
(If that is not the official way to buy stickers, I don't know where my money went, but it should have been yours ;) )

**Comment by friep** [comment]
I also ordered from redbubble because it seemed legit ("RStudio Inc") but was disappointed: the stickers were too large (i just assumed "small" would be the usual hex sticker size.. my fault) and quality was not nice. if this redbubble account is affiliated with RStudio, I'd recommend they take it down: People will have high expectations and will be disappointed (plus, i'm not even sure I'll be able to get a refund for my order). 
in summary, it seems there is no way to get "nice" stickers except ordering from stickermule in packs of 50. 😢

**Comment by burgerga** [comment]
@friep Sorry to hear about your experience, I ordered about 20 for our department and I've had no complaints, and I rather liked them!

**Comment by friep** [comment]
that's nice to hear! :)  i mean i could do with the quality but the "small" size is 7.6 by 8 cm which is really not the "typical" hex sticker you'd put on your laptop. 😂 but anyhow, just another reason to attend more R events! :)

**Comment by hadley** [comment]
We're going to pull them because they're the wrong size — we're continuing to look for some easier for people to get small quantities of individual stickers; hopefully will have something early next year.
